jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -249,7 +249,7 @@ frontend http-in
   <%- end -%>
   <%- if_p("ha_proxy.rsp_headers") do |rsp_headers| -%>
     <%- rsp_headers.each do |rsp_header, value| -%>
-    http-response add-header <%= header.gsub(/(?!:\\)( )/, '\ ') %> "<%= value.to_s.gsub(/(?!:\\) /, '\ ') %>"
+    http-response add-header <%= rsp_headers.gsub(/(?!:\\)( )/, '\ ') %> "<%= value.to_s.gsub(/(?!:\\) /, '\ ') %>"
```

Comment:
hi @ogrand good catch! though there are 2 more occasions of this issue.
Can you do me a favor and update your PR to also include these locations:
[Line 335](https://github.com/cloudfoundry-incubator/haproxy-boshrelease/pull/171/files#diff-f97ebf501b3e4f8aab101b7981a63562c73d6ffcd6e133ab48735c9880a2043cR335)

[Line 411](https://github.com/cloudfoundry-incubator/haproxy-boshrelease/pull/171/files#diff-f97ebf501b3e4f8aab101b7981a63562c73d6ffcd6e133ab48735c9880a2043cR411)

Also please sign the CLA so I can merge your PR. Thanks!

Reply:
hi @domdom82, I have fixed the other mismatch.
My organisation will try to end CLA process in easyCLA. Hope that it will done quickly.
Regards.

Reply:
hi @ogrand I just saw another typo in your fix:
`rsp_headers ` should be `rsp_header` as it is the loop variable for a single header. :)